scripts/run-kani.sh

```
@@ -0,0 +1,36 @@
+    done
+}
+
+if [ $# -eq 0 ]; then
+    run_clone_detector_harnesses
+    exit 0
+fi
+
+case "$1" in
+    clone-detector)
+        shift
+        run_clone_detector_harnesses
```

Shellcheck (0.11.0)

[info] Use run_clone_detector_harnesses "$@" if function's $1 should mean script's $1.

(SC2119)